frontend/.env.example

```diff
@@ -0,0 +1,3 @@
+EIFFELVIS_URL = "@hostname:3001"
```

Comment:
is this relevant to the PR?

Reply:
No! It slipped in during cherry picking, will change that! Thanks for pointing it out 😃